api/src/test/java/org/openmrs/module/mirebalais/printer/impl/ZlEmrIdCardPrinterTest.java

```diff
@@ -87,11 +83,16 @@ public void setup() throws Exception {
         PrinterModuleActivator printerModuleActivator = new PrinterModuleActivator();
         printerModuleActivator.started(); // Create Location Attribute Types Needed
 
-        haitiMetadataBundle.install(); // to install primary identifier type
-        pihHaitiPatientIdentifierTypeBundle.install(); // Install Patient Identifier Types for distribution
         personAttributeTypeBundle.install(); // Install Person Attribute Types for distribution
         haitiPersonAttributeTypeBundle.install(); // Instal Person Attribute Types provided by Haiti Core
 
+        // TODO: This might not work any more after removing the bundles.  I added this in but might need to be tweaked to make it actually work
+        PatientIdentifierType zlemrId = new PatientIdentifierType();
+        zlemrId.setUuid(ZlConfigConstants.PATIENTIDENTIFIERTYPE_ZLEMRID_UUID);
+        zlemrId.setName("ZL EMR ID");
+        zlemrId.setLocationBehavior(PatientIdentifierType.LocationBehavior.NOT_USED);
+        patientService.savePatientIdentifierType(zlemrId);
```

Comment:
Can you run this and add a breakpoint in ZlEmrIdCardPrinter to see if all the parameters get properly copied in: https://github.com/PIH/openmrs-module-mirebalais/blob/master/api/src/main/java/org/openmrs/module/mirebalais/printer/impl/ZlEmrIdCardPrinter.java#L121

Reply:
Yes, but I'm going to merge this in first, and do that second.